rstudio/c9s-python-3.9/Dockerfile

```
@@ -45,6 +45,9 @@ RUN chmod 1777 /var/run/rstudio-server && \
     mkdir -p /usr/share/doc/R
 COPY rsession.conf /etc/rstudio/rsession.conf
 
+# package installation 
+RUN  dnf install -y libsodium-devel.x86_64 && dnf install -y libgit2-devel.x86_64 && dnf install -y libcurl-devel && dnf install -y cmake && dnf install -y flexiblas-*
+
```

Could you please add also this line on the Dockerfile? 

```suggestion
RUN R -e "install.packages('Rcpp')"
```

This package has to be installed first, execute it on its own line. It is the C compiler for R. If this is missing, the majority of the packages will fail installing, either directly or due to missing dependencies. 

This will solve also the issue with the Torch package installation.